docs/source/config/authentication.rst

```diff
@@ -216,14 +259,55 @@ Example ``auth`` section in the |st2| configuration file. ::
     logging = /path/to/st2auth.logging.conf
     api_url = http://myhost.example.com:9101/
 
-After the configuration change, restart all st2 components. ::
+LDAP backend
+~~~~~~~~~~~~
+
+Repository URL: https://github.com/StackStorm/st2-auth-backend-ldap
+
+Backend which reads authentication information from an LDAP server. The backend tries to bind the
+ldap user with given username and password. If the bind was successful, it tries to find the user
+in the given group. If the user is in the group, they will be authenticated.
+
+**Backend configuration options:**
+
+* ``ldap_server`` - URL of the LDAP Server.
+* ``base_dn`` - Base DN on the LDAP Server.
+* ``group_dn`` - Group DN on the LDAP Server which contains the user as member.
+* ``scope`` - Scope search parameter. Can be base, onelevel or subtree (default: subtree)
+* ``use_tls`` - Boolean parameter to set if tls is required
+* ``search_filter`` - Should contain the placeholder %(username)s for the username. (Default:
+  ``uniqueMember=uid=%(username)s,base_dn``)
+
+Example ``auth`` section in the |st2| configuration file.
+
+.. sourcecode:: ini
+
+    [auth]
+    mode = standalone
+    backend = ldap_backend
+    backend_kwargs = {"ldap_server": "ldap://ds.example.com", "base_dn": "ou=people,dc=example,dc=com", "group_dn": "cn=sysadmins,ou=groups,dc=example,dc=com", "scope": "", "use_tls": True}
```

Comment:
You are a bit late to the game. The way config options are specified didn't change :P

We can tackle this later, if needed.